README.md

```
@@ -1 +1,118 @@
+
+## npm 스크립트
+
+| 스크립트 | 설명 |
+| --- | --- |
+| `npm run dev` | 개발 서버 실행 (0.0.0.0:5173) |
+| `npm run build` | `tsc -b` 타입 체크 후 프로덕션 빌드 |
+| `npm run preview` | 빌드 결과물 정적 서빙 (0.0.0.0:4173) |
+
+## 디렉터리 구조
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)